subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/DeviceWorker.kt

```diff
@@ -36,7 +36,7 @@ class DeviceWorker(
             checkDeviceAlive()
             device.state()
         } catch (t: Exception) {
-            device.notifyError("Error while checking device initial state", t)
+            device.warn("Error while checking device initial state", t)
```

Comment:
`notifyError` were pointed to `CILogger.critical`.

As discussed with @sboishtyan , it can't be critical. This is an expected and frequent event. 
To trace stability and performance issues we should use metrics but not errors.
Errors in Sentry must be actionable and closeable.